scripts/invoke_view_function.ts

```diff
@@ -12,8 +10,14 @@ module.exports = async (callback: Truffle.ScriptCallback) => {
     }
     const [functionName, ...arg] = args;
     const exchange = await getBatchExchange(artifacts);
-    const info = await (exchange as any)[functionName].call(...arg);
-    log.info(info);
+    const info: unknown = await exchange.contract.methods[functionName](
+      ...arg,
+    ).call();
+    // Note that this script is intended to be used for bash scripting and
+    // one would usually grep for the results of this execution. Thus,
+    // the raw output is printed rather than include irrelevant log details.
+    // eslint-disable-next-line no-console
+    console.log(info);
```

Comment:
Curious, is there a reason you didn't use `info.toString()` ?

Reply:
Actually I think I meant to...
```suggestion
    console.log(info.toString());
```